chips/nrf52/src/spi.rs

```diff
@@ -0,0 +1,501 @@
+use chip;
```

Comment:
We usually like a module comment at the top (e.g. https://github.com/helena-project/tock/blob/master/chips/sam4l/src/adc.rs#L1). Things worth including:
  - what is it (usually an obvious one-liner)
  - how complete is the implementation / any caveats
  - an author line
  - a last (major) update date

Reply:
Done.